src/components/ContentSidebar/ActivityFeed/task/__tests__/Task-test.js

```diff
@@ -89,13 +90,13 @@ describe('components/ContentSidebar/ActivityFeed/task/Task', () => {
             isPending: true,
         };
 
-        const wrapper = shallow(<Task currentUser={currentUser} {...myTask} />);
+        const wrapper = shallow(<Task currentUser={currentUser} onEdit={jest.fn()} onDelete={jest.fn()} {...myTask} />);
         expect(wrapper.hasClass('bcs-is-pending')).toBe(true);
     });
 
     test('should show actions for current user and if onAssignmentUpdate is defined', () => {
         task.isPending = false;
-        const wrapper = shallow(<Task currentUser={currentUser} {...task} onAssignmentUpdate={jest.fn()} />);
+        const wrapper = mount(<Task currentUser={currentUser} {...task} onAssignmentUpdate={jest.fn()} />);
```

Comment:
Is mount giving us anything over shallow here?

Reply:
Hmm, this appears to be a leftover from trying to get more visibility into the `<mock-comment` being rendered. I settled for ensuring that the props were being passed to `<Comment` and then making sure that the tests on comment were doing their job.